spec/betterc.dd

```diff
@@ -0,0 +1,48 @@
+Ddoc
+
+$(SPEC_S Better C,
+
+`-betterC` is a Command-Line flag for `dmd`, which generated light-weight,
+barebone, runtime-free binaries with are suitable for linking from C.
```

Comment:
I'm not sure either, but one simple litmus test is that the compiler still calls D applications' `main` from `extern (C) main` , as opposed to `_start` (on Linux). I.e. it expects the C runtime to perform its initialization.